app/imports/api/users/methods.js

```diff
@@ -38,3 +40,60 @@ export const removeAdmin = new ValidatedMethod({
     },
 });
 
+export const setProfileName = new ValidatedMethod({
+    name: 'users.setName',
+    validate: setNameSchema.validator({ clean: true }),
+    run({ name }) {
+        const userId = Meteor.userId();
+
+        if (userId) {
+            return Meteor.users.update(userId, {
+                $set: {
+                    'profile.name': name,
+                },
+            });
+        }
+        return undefined;
+    },
+});
+
+export const setGravatarAvatar = new ValidatedMethod({
+    name: 'users.setGravatar',
+    validate: new SimpleSchema({}).validator({}),
+    run() {
+        const user = Meteor.user();
+
+        if (user) {
+            const userId = user._id;
+            const email = user.services.github.email;
+            const emailHash = md5(email);
+
+            return Meteor.users.update(userId, {
+                $set: {
+                    'profile.avatar': `https://s.gravatar.com/avatar/${emailHash}`,
+                },
+            });
+        }
+        return undefined;
+    },
+});
+
+export const setGithubAvatar = new ValidatedMethod({
```

Comment:
I think that one method would be better, just `setAvatar`, you can use your functions inside ;)